toloka_provider/utils/utils.py

```diff
@@ -0,0 +1,54 @@
+"""This module contains util functions."""
+import json
+import os
+import pickle
+from decimal import Decimal
+from functools import wraps
+from typing import Any, Type, TypeVar
+
+from toloka.client import structure, add_headers
+
+T = TypeVar('T')
+
+
+def serialize_if_needed(func):
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        default_xcom_backend = 'airflow.models.xcom.BaseXCom'
+        xcom_backend = os.getenv('AIRFLOW__CORE__XCOM_BACKEND', default_xcom_backend)
+        need_to_serialize = (xcom_backend == default_xcom_backend)
+        result = add_headers('airflow')(func)(*args, **kwargs)
+        if need_to_serialize and result is not None:
```

Comment:
Looks like need_to_serizlize is if_default_backend. Meybe we should really consider renaming the decorator

Reply:
rename to `serialize_if_default_xcom`